go/store/nbs/manifest.go

```diff
@@ -468,10 +469,10 @@ func parseSpecs(tableInfo []string) ([]tableSpec, error) {
 	specs := make([]tableSpec, len(tableInfo)/2)
 	for i := range specs {
 		var err error
```

Comment:
Nit: This seems to be unused now...

Reply:
The function? It's used twice in file_manifest.go and twice in dynamo_manifest.go